src/main/java/ru/my/Mocito/employeeService/EmployeeImpl.java

```diff
@@ -14,27 +14,27 @@
 public class EmployeeImpl implements Iemployee {
     private final Map<String, Employee> employees = new HashMap<>();
 
-    final int maxEmployee = 13;
+    private final int maxEmployee = 13;
 
     @Override
-    public String addEmployee(String fullName, int department, int salary) {
+    public Employee addEmployee(String fullName, int department, int salary) {
         Employee employee = new Employee(fullName, department, salary);
         if (employees.size() >= maxEmployee) {
             throw new EmployeeStorageIsFullException("свободных мест нет");
         } else if (employees.containsKey(employee.getFullName())) {
             throw new EmployeeAlreadyAddedException("такой сотрудник есть");
         }
         employees.put(fullName, employee);
-        return employee.toString();
+        return employee;
     }
 
     @Override
-    public String removeEmployee(String fullName) {
+    public Employee removeEmployee(String fullName) {
         if (!employees.containsKey(fullName)) {
             throw new EmployeeNotFoundException("сотрудника нету");
         } else {
             employees.remove(fullName);
-            return fullName + " сотрудник удален";
+            return employees.get(fullName);
```

Comment:
сотрудник удалён выше уже. Надо либо достать ДО удаления и после удаления вернуть из метода. Либо ещё проще - метод remove у карты уже возвращает удаленное значение, т.е. просто
```suggestion
            return employees.remove(fullName);
```